File: andy-manager-web/src/main/webapp/js/service/brandService.js
//定义品牌服务
		app.service("brandService",function($http){
			//模型变量绑定视图,剩余部分属于控制层代码,要拿结果response,
			//和后端交互部分
			this.findAll=function(){
				return $http.get('..brand/findAll.do');
			}
			this.findPage=function(page,sizess){
				return $http.get('../brand/findPage.do?page='+page+'&size='+sizess);
			}
			this.findOne=function(id){
				return $http.get('../brand/findOne.do?id='+id);
			}
			
			this.add=function(entity){
				return $http.post('../brand/add.do',entity);
			}
			this.update=function(entity){
				return $http.post('../brand/update.do',entity);
			}
			this.dele=function(ids){
				return $http.get('../brand/delete.do?ids='+ids);
			}
			this.search=function(page,sizess,searchEntity){
				return $http.post('../brand/search.do?page='+page+'&size='+sizess,searchEntity);
			}
			
			//下拉商品选项
			this.selectOptionList=function(){
				return $http.get('../brand/selectOptionList.do');
			}
			
			
			
		});

File: andy-sellergoods-interface/src/main/java/com/andy/sellergoods/service/BrandService.java
package com.andy.sellergoods.service;


import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

/**
 * 品牌接口
 * @author 李星
 *
 */
public interface BrandService {
	public List<TbBrand> findAll();
	/**
	 * 品牌分页
	 * @param pageNum 当前页面页码
	 * @param pageSize每页记录数
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 增加品牌
	 */
	public void add(TbBrand tbBrand);
	/**
	 * 根据id查询
	 * @param id
	 * @return
	 */
	public TbBrand findOne(long id);
	/**
	 * 修改
	 * @param tbBrand
	 */
	public void update(TbBrand tbBrand);
	/**
	 * 删除
	 * @param ids
	 */
	public void delete(long[] ids);
	/**
	 * 条件分页
	 * @param tbBrand
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);
	/**
	 * 品牌下拉框数据
	 */
	public List<Map> selectOptionList();
	
}
